app/code/Magento/Inventory/Model/ResourceModel/Source/Type.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model\ResourceModel\Source;
+
+use Magento\Framework\Model\ResourceModel\Db\AbstractDb;
+
+class Type extends AbstractDb
```

Comment:
Some sort of the docBlock with description of the class purpose would be really helpful here